pkg/flow/logging/logger.go

```diff
@@ -126,9 +126,9 @@ func (l *Logger) Update(o Options) error {
 
 	// Print out the buffered logs since we determined the log format already
 	for _, bufferedLogChunk := range l.buffer {
-		if err := slogadapter.GoKit(l.handler).Log(bufferedLogChunk...); err != nil {
-			return err
-		}
+		// the buffered logs are currently only sent to the standard output
+		// because the components with the receivers are not running yet
```

Comment:
Can we rephrase that comment? It is an explanation of why we ignore the error, but isn't true for the entire rationale behind l.buffer.

Reply:
it's true for the entire lifetime of the buffer because it's set to nil and wont be used anymore after this

Reply:
> does it mean that the buffered logs will never reach a loki.write receiver, for example? 

yes because the logs are only buffered till the config node is evaluated, after that the logs are not buffered anymore. And no component can run before all nodes have been through the first evaluation cycle